readme.didact.md

```diff
@@ -60,7 +60,7 @@ In order to execute this demo, you will need to have an OpenShift cluster with t
 
 Apart from the support provided by the VS Code extension, you also need the Apache Camel K CLI ("kamel") in order to access all Camel K features.
 
-[Check if the Apache Camel K CLI ("kamel") is installed](didact://?commandId=vscode.didact.requirementCheck&text=kamel-requirements-status$$kamel%20version$$Camel%20K%20Client&completion=Apache%20Camel%20K%20CLI%20is%20available%20on%20this%20system. "Tests to see if `kamel version` returns a result"){.didact}
+[Check if the Apache Camel K CLI ("kamel") is installed](didact://?commandId=vscode.didact.cliCommandSuccessful&text=kamel-requirements-status$$kamel%20version$$Camel%20K%20Client&completion=Apache%20Camel%20K%20CLI%20is%20available%20on%20this%20system. "Tests to see if `kamel version` returns a result"){.didact}
```

Comment:
This one can stay as `requirementCheck`, it's the one above (`Check if you're connected to an OpenShift cluster`) that needs to be `cliCommandSuccessful`

Reply:
Yeah, I am not very familiar with their difference, but I understood the `cliCommandSuccessful` is looking for a 0 return from the command.